### ISSUE TITLE
Have "related" use page title and/or bracket title so it acts as a link

### ISSUE THREAD

**Issue by cobblepot1** [melat0nin/zotero-roam-export#33]
I exported a book and all chapters as separate related items into Roam. I use the BibTex citekey page title option, but items listed as "related" are listed with the normal title, so they don't link to the correct source pages. The best solution for me would be to have the "related" items listed with the Roam page titles (either citekey or normal title). If that's not possible, could you at least wrap the title line (2nd metadata item) in brackets so it acts like a link? I'll still have to make a manual link, but it will be easier to find the correct page. Many thanks!

**Comment by melat0nin** [comment]
To clarify, when the citekey-as-title option is enabled, the natural language title in the metadata should be [[linked]]? That's simple enough to do, but what's the most useful approach -- linking the natural language title or providing a link to the citekey? Presumably those who are relying on citekey-as-title as part of their workflow would prefer the latter?

If you can create a mockup of what exactly you're imagining that would be helpful, just so I'm clear before I implement anything.

**Comment by cobblepot1** [comment]
Consider two items: a book and a section in that book:

- Green, M. C., Strange, J. J., & Brock, T. C. (Eds.). (2002). Narrative impact: Social and cognitive foundations. Lawrence Erlbaum Associates.

- Graesser, A. C., Olde, B., & Klettke, B. (2002). How does the mind construct and represent stories? In M. C. Green, J. J. Strange, & T. C. Brock (Eds.), Narrative impact: Social and cognitive foundations (pp. 229–262). Lawrence Erlbaum Associates.

Currently, if I export these to roam, the book page is titled `@greenNarrativeImpactSocial2002` and that page includes this: 
```
Related:
     - [[How does the mind construct and represent stories?]]
```

But the Roam page that is created when I export that book section is not titled `How does the mind construct and represent stories?`, it is actually titled `@graesserHowDoesMind2002`. So ideally, when page titles are citekeys, the page titles in the "Related" section will also use citekeys, so the page for `@greenNarrativeImpactSocial2002` should instead include this: 

```
Related:
     - [@graesserHowDoesMind2002]]
```

Does that make sense?

**Comment by melat0nin** [comment]
> 
> 
> Consider two items: a book and a section in that book:
> 
>     * Green, M. C., Strange, J. J., & Brock, T. C. (Eds.). (2002). Narrative impact: Social and cognitive foundations. Lawrence Erlbaum Associates.
> 
>     * Graesser, A. C., Olde, B., & Klettke, B. (2002). How does the mind construct and represent stories? In M. C. Green, J. J. Strange, & T. C. Brock (Eds.), Narrative impact: Social and cognitive foundations (pp. 229–262). Lawrence Erlbaum Associates.
> 
> 
> Currently, if I export these to roam, the book page is titled `@greenNarrativeImpactSocial2002` and that page includes this:
> 
> ```
> Related:
>      - [[How does the mind construct and represent stories?]]
> ```
> 
> But the Roam page that is created when I export that book section is not titled `How does the mind construct and represent stories?`, it is actually titled `@graesserHowDoesMind2002`. So ideally, when page titles are citekeys, the page titles in the "Related" section will also use citekeys, so the page for `@greenNarrativeImpactSocial2002` should instead include this:
> 
> ```
> Related:
>      - [@graesserHowDoesMind2002]]
> ```
> 
> Does that make sense?

Yep that makes sense -- in other words, when citekey-as-title is enabled, it should apply to the related items' titles as well as the item actually being exported. I will take a look.

(Interesting literature, by the way!)
